pkg/types/machinepools.go

```diff
@@ -50,6 +50,6 @@ type EC2RootVolume struct {
 // LibvirtMachinePoolPlatform stores the configuration for a machine pool
 // installed on libvirt.
 type LibvirtMachinePoolPlatform struct {
-	// QCOWImagePath
-	QCOWImagePath string `json:"qcowImagePath"`
+	BaseImagePool   string `json:"baseImagePool"`
+	BaseImageVolume string `json:"baseImageVolume"`
```

Comment:
While we're touching these, I'm in favor of just using `Image` (with `json:"image"`) here too.